Makefile

```diff
@@ -76,3 +80,4 @@ clean:
 	rm -rf dist
 	python setup.py clean
 	rm -rf site
+	docker-compose down -v
```

Comment:
It seems to me that it is better to put this in a separate command (as with `up-services`), adding it here as a dependency (as in tests).

Like:
```make
.PHONY: down-services
down-services:
    docker-compose down -v
...
clean: down-services
...
```